web/src/components/core/IssuesDialog.jsx

```diff
@@ -78,39 +77,53 @@ const IssueItems = ({ issues = [] }) => {
  *
  * @param {object} props
  * @param {import ("~/client/issues").ClientsIssues} props.issues
+ * @param {string} [props.openFrom] - A string which indicites which issues section should be highlighted.
  */
-const IssuesSections = ({ issues }) => {
+const IssuesSections = ({ issues, openFrom = "" }) => {
```

Comment:
In general, I think we can go more _aggressive_ here and drop all the logic for building sections conditionally. I mean, the `IssuesDialog` should receive the `key`, `source`, or similar prop and retrieve the issues only for such "key", "source", "section", whatever. Thus, the IssuesDialog can be internally simplified a lot, just as we are doing visually.


```jsx

const issues = issues[key] || [];

...

return  issues.length ? <IssuesList /> : <NoIssuesFound>

```
No _stack_ needed, no `<If>`s needed, no `<Section>`'s `title` needed (but please, still using a `<Section>` for wrapping the content).

Let me know if you get what I mean.

---

// cc @imobachgs maybe the `IssuesDialog` title should be given via prop and should be something like "Software issues", "Storage issues" or "Software problems", "Storage problems", or so. Again, we need your feedback here.

Reply:
Using "Software issues", "Storage issues", and so on looks good enough, although I think we should discuss whether "issue" is a good term. But we can postpone that.